.expeditor/config.yml

```diff
@@ -78,9 +78,9 @@ github:
    - "Expeditor: Bump Major Version"
 
 release_branches:
-  - inspec-6:
```

Comment:
At this time, we should not delete the inspec-6 release branch from the configuration. We want to leave it untouched, matching version 6.* .